packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -46,7 +44,7 @@ export default async function (rta: RuntimeAuthoring) {
         }
     );
 
-    const syncViewsIds = getAllSyncViewsIds();
+    const syncViewsIds = await getAllSyncViewsIds(minor);
```

Comment:
This seems to be executed before the app is completely loaded which means that maybe some of the views are not loaded. But this is not related to this particular fix.